### DISCUSSION TITLE
Problem with directional pad.

### DISCUSSION THREAD

**Discussion by cocori002** [mackieks/MaplePad#29] [General]
Hello!
I soldered everything to a dreamcast controller and all is good but have a little problem, when I'm in a menu the pad moves to de left all the time...

Anyone knows the solution or the problem?

Thanks.

**Comment by programmerta** [comment]
Sounds like the d-pad left is being pulled low as if it was being pressed. Make sure that your button is not shorted to ground.

**Comment by mackieks** [comment]
This is also common behavior when the triggers and/or joystick aren't calibrated properly. 

See this comment for potential solutions: https://github.com/mackieks/MaplePad/issues/21#issuecomment-2329176937
